tests/foreman/maintain/test_backup_restore.py

```diff
@@ -446,8 +404,8 @@ def test_positive_puppet_backup_restore(
 @pytest.mark.parametrize('backup_type', ['online', 'offline'])
 def test_positive_backup_restore(
```

Comment:
I do not understand the changes here.

Reply:
I think it should have been `module_target_sat` from the start, as when I compared `session_target_sat` and `sat_maintain`, they were different.

Reply:
```python
def sat_maintain(request, module_target_sat, module_capsule_configured):
```
And since this test is using `sat_maintain` fixture, and we want the test to point to the same Satellite and Capsule object, the test needs to use the same fixture as the `sat_maintain` fixture does.